hardware/tests/test_benchmark_mem.py

```diff
@@ -145,18 +123,6 @@ def test_run_sysbench_memory_forked_bytes(self, mock_popen,
         self.assertEqual([('cpu', 'logical', 'forked_bandwidth_1K', '382')],
                          hw_data)
 
-    def test_run_sysbench_memory_threaded_text(self, mock_popen,
```

Comment:
ditto

Reply:
because the output of commands in Python 3.x is bytes, not pure text, so we don't need this, we already have the correspondent bytes test